src/utils/positions.ts

```
@@ -170,6 +243,7 @@ export async function calculateEarningsFromPeriod(
   userAddress: Address,
   chainId: SupportedNetworks,
   blockNumbers: { day: number; week: number; month: number },
+  customRpcUrl?: string,
```

ESLint

[error] 'customRpcUrl' is defined but never used.

(@typescript-eslint/no-unused-vars)